src/ucp/proto/proto_common.inl

```diff
@@ -80,7 +80,7 @@ ucp_proto_request_zcopy_cleanup(ucp_request_t *req, unsigned dt_mask)
 static UCS_F_ALWAYS_INLINE void
 ucp_proto_request_zcopy_complete(ucp_request_t *req, ucs_status_t status)
 {
-    ucp_proto_request_zcopy_cleanup(req, UCP_DT_MASK_CONTIG_IOV);
+    ucp_proto_request_zcopy_cleanup(req, UCP_DT_MASK_ALL);
```

Comment:
seems weird. zcopy should not be used for generic type.

Reply:
I can change it to be a parameter. We have to handle a case when ATS is received for generic dt send, because of truncation (this func is called from `ucp_proto_rndv_ats_handler`).
BTW, [here](https://github.com/openucx/ucx/blob/master/src/ucp/proto/proto_common.c#L786) "all" mask is already used in zcopy completion routine

Reply:
the other place seems like error as well. or, at least, wrong name for this function.
can we not call ucp_proto_request_zcopy_complete from ucp_proto_rndv_ats_handler? since now that we changed RTS selection logic, ATS can complete also a non-copy request.